gym_sts/data/state_logger.py

```diff
@@ -29,12 +27,13 @@ def log(self, action: Optional[Action], after_obs: Observation):
             self.flush_actions()
 
     def flush_actions(self):
-        now = datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
-        with open(self.logdir / f"states_{now}.json", "w") as f:
-            f.write(json.dumps(self.unlogged_actions))
+        # Uncomment next time we want to collect more data
+        # now = datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
+        # with open(self.logdir / f"states_{now}.json", "w") as f:
+        #    f.write(json.dumps(self.unlogged_actions))
 
         self.unlogged_actions = []
 
         # TODO: Implement writing to WandB
 
-        print("Actions flushed.")
+        print("Actions not being logged.")
```

Comment:
Can we make this configurable in the env constructor instead? I've depended on this logging being on by default.

Reply:
I added a log_states argument.